package.json

```diff
@@ -19,10 +19,16 @@
     "homepage": "https://next.catcolab.org",
     "devDependencies": {
         "@biomejs/biome": "^1.8.3",
+        "autoprefixer": "^10.4.20",
         "biome": "^0.3.3",
         "depcheck": "^1.4.7",
+        "postcss": "^8.4.47",
+        "tailwindcss": "^3.4.13",
```

Comment:
Using tailwind is a significant decision. I don't actually see it used anywhere here, and I'm doubtful that we need it. Remove this too?